.circleci/config.yml

```diff
@@ -1678,6 +1678,196 @@ jobs:
             MESSAGE: << parameters.message >>
             LABEL_NAME: << parameters.label_name >>
 
+  workflow-completed:
+    docker:
+      - image: <<pipeline.parameters.default_docker_image>>
+    steps:
+      - run:
+          name: Workflow Completed
+          command: echo "Workflow completed successfully!"
+
+  kurtosis-pr-sanity-check:
+    docker:
+      - image: cimg/go:1.21
+    resource_class: medium
+    steps:
+      - checkout
+      - run:
+          name: Install Kurtosis
+          command: |
+            echo "Installing Kurtosis CLI..."
+            curl -sSL https://get.kurtosis.com/install.sh | bash
+            echo 'export PATH="$HOME/.kurtosis/bin:$PATH"' >> $BASH_ENV
+            source $BASH_ENV
+            kurtosis version
+      - run:
+          name: Start Kurtosis Engine
+          command: |
+            kurtosis engine start
+            kurtosis engine status
+      - run:
+          name: Build Required Docker Images
+          working_directory: kurtosis-devnet
+          command: |
+            # Install just
+            curl --proto '=https' --tlsv1.2 -sSf https://just.systems/install.sh | bash -s -- --to ~/.local/bin
+            export PATH="$HOME/.local/bin:$PATH"
+
+            # Build the minimal set of Docker images required for the check
```

Comment:
Is there a particular reason for building these upfront?
`just simple-devnet` will build what's necessary, so we shouldn't have to encode knowledge of these dependencies here (they might change)